### DISCUSSION TITLE
Please help me identify the font used?

### DISCUSSION THREAD

**Discussion by VPraharsha03** [JanDeDobbeleer/oh-my-posh#2276] [Troubleshoot] [answered]
Sorry for putting this up here, but I couldn't find the font being used in these themes:

![image](https://user-images.githubusercontent.com/56556436/168568745-12af6d2e-f52f-4983-9fb4-031802dbe341.png)

![image](https://user-images.githubusercontent.com/56556436/168568772-430c6a16-bfcb-47b4-b798-b30cc87457c2.png)

Meslo LGM NF doesn't work for me.. can anyone please list them?

**Comment by lewis-yeung** [comment, marked as answer]
@VPraharsha03 The font used by the theme renderer is [Hack Nerd Font](https://github.com/ryanoasis/nerd-fonts/releases/download/v2.1.0/Hack.zip). Preview available [here](https://www.programmingfonts.org/#hack).

**Comment by VPraharsha03** [comment]
@lewis-yeung Thank you. I've one more issue, WSL segment doesn't display properly for me, any idea on what's the issue?

![bash](https://user-images.githubusercontent.com/56556436/168588950-ca9f3c18-200e-4243-b742-1c2ff1e9d603.png)

**Comment by lewis-yeung** [comment]
@VPraharsha03 Do you mean that it doesn't show the WSL distro icon but a generic Linux one, and the icon isn't displayed completely? It seems that you are using a [clean-detailed](https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/themes/clean-detailed.omp.json) theme in bash. The issue doesn't appear on my end. OMP gets the WSL distro name/icon by reading the value of the environment variable `WSL_DISTRO_NAME`. Can you provide the debug log output by running `oh-my-posh debug`?

**Reply by VPraharsha03** [reply]
@lewis-yeung 
[omp.log](https://github.com/JanDeDobbeleer/oh-my-posh/files/8700095/omp.log)

**Comment by VPraharsha03** [comment]
Found anything? @lewis-yeung

**Comment by lewis-yeung** [comment]
@VPraharsha03 The value of the env `WSL_DISTRO_NAME` on your WSL is `Pengwin`:

```
2022/05/16 18:04:13 debug: Getenv
Pengwin
2022/05/16 18:04:13 Getenv duration: 5µs, args: WSL_DISTRO_NAME
```

For the time being, OMP doesn't have a predefined icon for this distro, so a generic "penguin" is used as the fall-back. (TBH, I think this icon is vivid enough. 😆) As for that the icon isn't displayed completely, it depends on the glyph width defined for a specific Nerd Font. You can tweak this by adding a space character after the icon manually, see [docs](https://ohmyposh.dev/docs/segments/os).

**Reply by lewis-yeung** [reply]
E.g.

```json
"properties": {
  "linux": "\uF17C "
}
```

**Comment by VPraharsha03** [comment]
😂 Thanks, will check that

Yes, adding the space worked
